api/src/services/application-csv-export.service.ts

```diff
@@ -313,10 +325,16 @@ export class ApplicationCsvExporterService
       {
         path: 'submissionType',
         label: 'Application Type',
+        format: (val: string): string =>
+          val === ApplicationSubmissionTypeEnum.electronical
```

Comment:
this is something to think about for the future:
why are we storing this as `electronical` in the first place, I think the front end also does some work the translate out of `electronical` 🤔